internal/servers/source/v0/source.go

```diff
@@ -101,15 +109,23 @@ func (s *SourceServer) Sync2(req *pb.Sync2_Request, stream pb.Source_Sync2Server
 	if err := dec.Decode(&spec); err != nil {
 		return status.Errorf(codes.InvalidArgument, "failed to decode spec: %v", err)
 	}
+	ctx := stream.Context()
+	if err := s.Plugin.Init(ctx, spec); err != nil {
```

Comment:
Was this missing or moved from someplace else?

Reply:
Yeah now we need to call it in `v0` as we don't want to break the v0 protocol